### ISSUE TITLE
Text editor on edit task is not rendered if task description with next code

### ISSUE THREAD

**Issue by vistar** [kanboard/kanboard#3763]
### Actual behaviour
Text editor on task edit is not rendered if task description with next code:
```
          'rel' => 'nofollow',
          'class' => 'use-ajax',
          'data-dialog-type' => 'modal',
          'data-dialog-options' => '{&quot;width&quot;: 700}',
```

Error in console:
`Uncaught SyntaxError: Unexpected token w in JSON at position 264
    at JSON.parse (<anonymous>)
    at Object.KB.render (app.min.js?1514340019:1)
    at i (app.min.js?1514340019:1)
    at d (app.min.js?1514340019:1)
    at app.min.js?1514340019:1
    at XMLHttpRequest.d.onreadystatechange (app.min.js?1514340019:1)`

### Expected behaviour
See task description.

### Steps to reproduce
1. Add task description for new task with code above.
2. Edit task with quick edit or from full view of task.

### Screenshots
https://i.imgur.com/Qz2OYF0.png

### Configuration

- Kanboard version: 1.2.0
- Database type and version: 10.0.32-MariaDB-0+deb8u1
- PHP version: 7.0.27
- OS: Debian 8
- Browser: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/65.0.3325.146 Safari/537.36

**Comment by fguillot** [comment]
It should be fixed in the development version.

**Comment by fguillot** [comment]
This issue should be fixed in version 1.2.4.
